hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/impl/StateStoreFileBaseImpl.java

```diff
@@ -137,6 +149,8 @@ public abstract <T extends BaseRecord> BufferedWriter getWriter(
    */
   protected abstract String getRootDir();
 
+  protected abstract int getConcurrentFilesAccessNumThreads();
```

Comment:
Can we provide an implementation here and then just have one set of configs for the following two
```
FEDERATION_STORE_PREFIX + "driver.file.async.threads";
FEDERATION_STORE_PREFIX + "driver.fs.async.threads";
```
I'm okay with keeping them separate though if you have prefer that.

Reply:
I believe separate configs would be better since we anyways have different implementation for many of other methods too. Sounds good to keep it as is?

Reply:
Yeah, good to keep it as it is.